interface-definitions/include/dns/dynamic-service-host-name-server.xml.i

```diff
@@ -0,0 +1,34 @@
+<!-- include start from dns/dynamic-service-host-name-server.xml.i -->
+<leafNode name="host-name">
+  <properties>
+    <help>Hostname to register with Dynamic DNS service</help>
+    <constraint>
+        #include <include/constraint/host-name.xml.i>
+    </constraint>
+    <constraintErrorMessage>Host-name must be alphanumeric and can contain hyphens</constraintErrorMessage>
+    <multi/>
+  </properties>
+</leafNode>
+<leafNode name="server">
+  <properties>
+    <help>Server to be updated</help>
```

Comment:
Can we be more verbose here? `Dynamic DNS server to send updates to` or something similar to tell the user: "Hey - that's your remote to talk to when sending DDNS updates"